plutus-book/doc/01-preface.adoc

```diff
@@ -18,3 +18,8 @@ and understand the basics of Haskell programming prior to reading this book.
 
 Come and explore the world of Plutus and learn our new language through
 real life scenarios and sample code snippets!
+
+*Disclaimer:*
```

Comment:
This should be either a heading or a paragraph title, rather than just some formatted text.

Reply:
no problem